### PR TITLE
Error in leap year?

### PR DESCRIPTION
**Pull request by mickaellalande**
I've tried this script; however, it adds +1 to all months of the leap years. It sounds like an error, or I am wrong? So I wrote the condition "and month == 2" line 86 so that only the month of February gets +1.

 - [ ] Closes #xxxx
 - [ ] Tests added
 - [ ] Passes `black . && mypy . && flake8`
 - [ ] Fully documented, including `whats-new.rst` for all changes and `api.rst` for new API

### CODE REVIEW

**Comment by max-sixty** [comment]
Hi @mickaellalande — thanks for the PR, happy to have you as an xarray contributor!

I think you're right.

I guess we don't have tests for these...

@mickaellalande because our docs branch (`stable`) is a bit behind master, tests are failing. I pushed a change to your code that branched off master. Could you please change the 'base' of your PR to master? Let me know if that's unclear.

Could you also add a change & your name to whatsnew?

**Comment by keewis** [comment]
Everyone has to start somewhere :)

To change the base, see [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request). Also, now that @max-sixty added some commits, you will probably have to `git pull` once before adding more changes yourself (otherwise it complains that the remote has different commits).

`$repository/doc/whats-new.rst` is a file where all the changes are listed.

**Comment by mickaellalande** [comment]
I have no idea if I did the right thing... I changed the 'base' to master on this pull request and also on the repository : mickaellalande/xarray that was created. I don't know if had to do both? Let me know if I'm doing a mess... 

I also changed the whatsnew from that repository (mickaellalande/xarray) and wrote: "Fix leap year (http://xarray.pydata.org/en/stable/examples/monthly-means.html) by Mickaël Lalande." under "Documentation".

**Comment by keewis** [comment]
this looks good to me, but I don't see your commit changing `whats-new.rst`. Did you push that commit?

Ignore the failure on windows, that is unrelated to this PR.

**Comment by max-sixty** [comment]
You changed the base perfectly!

I don't see the whatsnew change though?

**Comment by keewis** [comment]
that did it. @max-sixty, what do you think?

**Comment by mickaellalande** [comment]
Ok I made it back to the branch "patch-1"

File: doc/whats-new.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ Bug fixes
 
 Documentation
 ~~~~~~~~~~~~~
-
+- Fix leap year (http://xarray.pydata.org/en/stable/examples/monthly-means.html) by `Mickaël Lalande <https://github.com/mickaellalande>`_.
```
Comment (dcherian) [inline doc/whats-new.rst]:
```suggestion
- Fix leap year condition in example (http://xarray.pydata.org/en/stable/examples/monthly-means.html) by `Mickaël Lalande <https://github.com/mickaellalande>`_.
```

**Comment by dcherian** [comment]
Great. Thanks @mickaellalande . Welcome to xarray!

**Comment by max-sixty** [comment]
Thanks @mickaellalande !

And @keewis for the mentoring!

**Comment by mickaellalande** [comment]
Thanks all of you guys to be patient and for your explanations :) Last question: can archive or delete the mickaellalande/xarray repository on my account now?

**Comment by keewis** [comment]
deleting won't affect this PR, so I'd say you are free to do what you want with your repository

**Pull request merged**